RELEASE.md

```diff
@@ -0,0 +1,94 @@
+  -   [mayadataio/kubera-propel-agent-controller](https://cloud.docker.com/u/mayadataio/repository/docker/mayadataio/kubera-propel-agent-controller)
+  -   [mayadataio/kubera-propel-iscsi-ubuntu](https://cloud.docker.com/u/mayadataio/repository/docker/mayadataio/kubera-propel-iscsi-ubuntu)
+  -   [mayadataio/kubera-propel-fio](https://cloud.docker.com/u/mayadataio/repository/docker/mayadataio/kubera-propel-fio)
+  -   [mayadataio/kubera-action](https://cloud.docker.com/u/mayadataio/repository/docker/mayadataio/kubera-action)
+
+- KUBERA-AUTH IMAGE
+  -   [mayadataio/kubera-auth](https://cloud.docker.com/u/mayadataio/repository/docker/mayadataio/kubera-auth)
+
+- AGENT-GATEWAY IMAGES
+  -   [mayadataio/gateway-server](https://cloud.docker.com/u/mayadataio/repository/docker/mayadataio/gateway-server)
+  -   [mayadataio/gateway-client](https://cloud.docker.com/u/mayadataio/repository/docker/mayadataio/gateway-client)
+
+- KUBERA-CORE-UI IMAGE
+  -   [mayadataio/kubera-core-ui](https://cloud.docker.com/u/mayadataio/repository/docker/mayadataio/kubera-core-ui)
+
+- KUBERA-CORE-SERVER IMAGE
+  -   [mayadataio/kubera-core-server](https://cloud.docker.com/u/mayadataio/repository/docker/mayadataio/kubera-core-server)
+
+### Kubera Enterprise Modules
+  - [mayadata-io/kubera-litmus](https://github.com/mayadata-io/kubera-litmus)
+  - [mayadata-io/kubera-metal](https://github.com/mayadata-io/kubera-metal)
+  - [mayadata-io/kubera-auth](https://github.com/mayadata-io/kubera-auth)
+  - [mayadata-io/agent-gateway](https://github.com/mayadata-io/agent-gateway)
+  - [mayadata-io/kubera-core-ui](https://github.com/mayadata-io/kubera-core-ui)
+  - [mayadata-io/kubera-core-server](https://github.com/mayadata-io/kubera-core-server)
+
+### Release Process (Needs to follow in the sequential manner)
+  - It starts with making release branch in each of modules
+    Note- kubera-core-server release branch should be created at last
+    
+  - Change `source_branch` from `master` to `release_branch` in `.circleci/config.yaml` in the release branch 
+    ```bash
+    else
+        tag="$CIRCLE_BRANCH-ci"
+        source_branch="master"
+        build_type="ci"
+    fi;
+    ```
+  - Make GitHub tags with the release version for each of modules
+    Note- kubera-core-server github tag should be created at last
+    
+  - Wait for the build to be finished in each of the modules
+  
+  - Releasing helm charts
+     - Make a branch out of the master 
+        Note: Don't name the checkout branch with the release version
+     - Change the `image.tag` to release version in `values.yaml`
+       ```yaml
+        image:
+            registry: docker.io
+            organization: mayadataio
+            tag: # RELEASE_VERION
+       ```
+     - Make a GitHub tag with the same release version from this branch
+       ```bash 
+       git tag -fa TechPreview-3 -m "release TechPreview-3"
+       git push origin TechPreview-3
+       ``` 
```

Comment:
we can add few lines about what the jenkins job does here, unless @atulabhi wants to add it in next PR.

Reply:
@vishnuitta Atul can add it later